test/Bridge.test.js

```diff
@@ -356,57 +356,47 @@ describe("Bridge", async function () {
     });
 
     describe("check execute transfer saves correct statuses", async function () {
-      let mockedSafe, newBridge;
-      beforeEach(async function () {
-        mockedSafe = await smock.fake("ERC20Safe");
-        const newBridgeFactory = await ethers.getContractFactory("Bridge");
-        newBridge = await newBridgeFactory.deploy(boardMembers, quorum, mockedSafe.address);
-        mockedSafe.transfer.returns(true);
-
-        await newBridge.unpause();
-      });
-
```

Comment:
why was this deleted ?

Reply:
because the test failed when trying to create the mock ERC20Safe. 
The complete error stack is this one:
```
 1) Bridge
       executeTransfer
         check execute transfer saves correct statuses
           "before each" hook for "returns correct statuses":
     TypeError: provider.init is not a function
      at getHardhatBaseProvider (node_modules/@defi-wonderland/smock/src/utils/hardhat.ts:32:18)
      at Function.create (node_modules/@defi-wonderland/smock/src/sandbox.ts:117:55)
      at init (node_modules/@defi-wonderland/smock/src/index.ts:27:27)
      at Object.fake (node_modules/@defi-wonderland/smock/src/index.ts:13:23)
      at Context.<anonymous> (test/Bridge.test.js:361:34)
      at processImmediate (node:internal/timers:466:21)
```